SubsTracker.API/GroupMembersController.cs

```
@@ -0,0 +1,56 @@
+using AutoMapper;
+using Microsoft.AspNetCore.Mvc;
+using SubsTracker.BLL.DTOs.User;
+using SubsTracker.BLL.DTOs.User.Create;
+using SubsTracker.BLL.DTOs.User.Update;
+using SubsTracker.BLL.Interfaces;
+
+namespace SubsTracker.API.Controllers;
+
+[ApiController]
+[Route("api/[controller]")]
+public class Controller(IGroupMemberService service, IMapper mapper) : ControllerBase
+{
+    [HttpGet("{id:guid}")]
+    public async Task<GroupMemberViewModel?> GetById(Guid id, CancellationToken cancellationToken)
```

Throw Not Found instead of null